internal/config/config.go

```diff
@@ -161,8 +166,8 @@ type AzureConfig struct {
 	//   Enable secure boot for VMs. If enabled, the OS image has to include a virtual machine guest state (VMGS) blob.
 	SecureBoot *bool `yaml:"secureBoot" validate:"required"`
 	// description: |
-	//   Expected value for the field 'idkeydigest' in the AMD SEV-SNP attestation report. Only usable with ConfidentialVMs. See 4.6 and 7.3 in: https://www.amd.com/system/files/TechDocs/56860.pdf
-	IDKeyDigest string `yaml:"idKeyDigest" validate:"required_if=EnforceIdKeyDigest true,omitempty,hexadecimal,len=96"`
+	//   List of accepted values for the field 'idkeydigest' in the AMD SEV-SNP attestation report. Only usable with ConfidentialVMs. See 4.6 and 7.3 in: https://www.amd.com/system/files/TechDocs/56860.pdf
+	IDKeyDigests Digests `yaml:"idKeyDigests" validate:"required_if=EnforceIdKeyDigest true,omitempty"`
```

Comment:
There is not validation happening when parsing the config now, or is there? I think its a nice UX to catch malformed values.

Reply:
No there us not. I guess the earliest point where a malformed value now shows up is `constellation init`. This is still somewhat early and can be retried without terminating the cluster (just the error message currently says otherwise).

Reply:
A user won't know that the error message is not correct in that case :D

Reply:
Found a good place to do validation. Re-implemented it.